src/saml2/saml.py

```diff
@@ -166,7 +166,9 @@ def set_type(self, typ):
 
         try:
             self.extension_attributes[XSI_TYPE] = typ
+            self.extension_attributes['xmlns:xs'] = XS_NAMESPACE
```

Comment:
https://github.com/rohe/pysaml2/blob/master/src/saml2/__init__.py#L386 does it make sense to use the setter?